packages/flutter_driver/lib/src/driver.dart

```diff
@@ -207,30 +208,30 @@ class FlutterDriver {
       });
     }
 
+    // Waits for a signal from the VM service that the extension is registered
```

Comment:
I just moved this out from the inner block, but the return string should be equal to the `_kFlutterExtensionMethod` constant